### DISCUSSION TITLE
Create a new channel like #hiring-job-board but for candidates

### DISCUSSION THREAD

**Discussion by smoya** [bcneng/candebot#94] [Ideas]
Imagine a channel like #hiring-job-board, with an automated posting system but for candidates that are looking for a job.
Candidates but also recruitment companies could post their profiles in there so companies who are hiring could check and if interested, contact them **somehow.**

WDYT?

**Comment by sdecandelario** [comment]
For me make quite sense, since sometimes you want to post that you are working for a job and there is no specific channel, also would be nice to allow freelance and contractors to post, since sometimes the channel about freelancers/contractors can be missed or the offer be lost in time

**Comment by alexpdp7** [comment]
I assume the purpose here is to help people looking for a job get one.

I think the question here is who would use the channel to look for candidates.

People do already post they are available in #hiring-discussions, but my general intuition is that really no one who is looking for candidates uses these posts. However, my intuition is frequently wrong, so I would look for a way to validate this hypothesis.

I am not good at that kind of thing, but what I've heard is that people tend to be optimistic/nice when polled for these kinds of questions. That is, people might answer "yes" to questions like "would you use a #hiring-looking channel to find candidates when hiring someone?" even when they would not do this in practice. I would find it more interesting to ask "have you used 'looking for a job' posts in #hiring-discussions to help when looking for someone to hire?"

This is a very personal opinion, and I am not close enough to people struggling with finding their first job, etc., to have a good feel for this, but IMHO, providing yet another thing that candidates "must" do to maximize their chances of landing a job seems... overburdening people who must already:

* Prepare a good CV and LinkedIn
* Develop personal projects to have a "nice GitHub"
* Go to meetups and network
* Do training
* Look for mentors
* Grind algorithms problems
* Prepare system design interviews
* ...

(In general, I feel landing a first job is an arms race- people find ways to stand out more, that are "published", and then everyone feels obliged to do more and more because everyone else is doing that. Not sure if this really a good take.)

(Also, the logical conclusion to what I'm saying is that if we think that #hiring-looking is not a good idea, then this kind of posts should also be forbidden in #hiring-discussions. But this is very cold-blooded, and I'm likely biased because these posts are not useful for me.)
